scripts/build_programs.py

```diff
@@ -116,5 +108,15 @@ def run_cmd(args) -> bool:
             "--zip",
             str(zip_path),
         ]
-    ), f"could not build double precision binaries: {dp_programs}"
+    ), f"could not build double precision binaries: {dp_exes}"
+    assert run_cmd(
+        [
+            "make-code-json",
+            "-ad", str(path),
+            "-f",
+            str(path / "code.json"),
+            "--verbose",
```

Comment:
The default '-f' is 'code.json' so you could exclude '-f str(path /"code.json"' if you want

Reply:
thanks, done (despite misnaming the argument in the commit message)